### ISSUE TITLE
Images should have an alt text

### ISSUE THREAD

**Issue by mkevenaar** [fdm-monster/fdm-monster-client#1771]
![image](https://github.com/user-attachments/assets/85ef8bd0-0998-4316-9302-e532a5516b85)

Images and buttons should all have descriptive alt texts. This improves FDM Monster for people who are blind and allow them to operate FDM Monster.
